dashboard/config/scripts/levels/Allthethings Java Lab 3.level

```diff
@@ -8,291 +8,7 @@
e":0,"assetId":0}],[{"tileType":1,"value":0,"assetId":0},{"tileType":1,"value":0,"assetId":0},{"tileType":1,"value":0,"assetId":0},{"tileType":1,"value":0,"assetId":0},{"tileType":1,"value":0,"assetId":0},{"tileType":1,"value":0,"assetId":0},{"tileType":1,"value":0,"assetId":0},{"tileType":1,"value":0,"assetId":0},{"tileType":0,"value":0,"assetId":285},{"tileType":1,"value":0,"assetId":0},{"tileType":1,"value":0,"assetId":0},{"tileType":1,"value":0,"assetId":0},{"tileType":1,"value":0,"assetId":0},{"tileType":1,"value":0,"assetId":0},{"tileType":1,"value":0,"assetId":0},{"tileType":1,"value":0,"assetId":0}],[{"tileType":1,"value":0,"assetId":0},{"tileType":1,"value":0,"assetId":0},{"tileType":1,"value":0,"assetId":0},{"tileType":1,"value":0,"assetId":0},{"tileType":1,"value":0,"assetId":0},{"tileType":1,"value":0,"assetId":0},{"tileType":1,"value":0,"assetId":0},{"tileType":1,"value":0,"assetId":0},{"tileType":0,"value":0,"assetId":285},{"tileType":1,"value":0,"assetId":0},{"tileType":1,"value":0,"assetId":0},{"tileType":1,"value":0,"assetId":0},{"tileType":1,"value":0,"assetId":0},{"tileType":1,"value":0,"assetId":0},{"tileType":1,"value":0,"assetId":0},{"tileType":1,"value":0,"assetId":0}],[{"tileType":1,"value":0,"assetId":0},{"tileType":1,"value":0,"assetId":0},{"tileType":1,"value":0,"assetId":0},{"tileType":1,"value":0,"assetId":0},{"tileType":1,"value":0,"assetId":0},{"tileType":1,"value":0,"assetId":0},{"tileType":1,"value":0,"assetId":0},{"tileType":1,"value":0,"assetId":0},{"tileType":0,"value":0,"assetId":285},{"tileType":1,"value":0,"assetId":0},{"tileType":1,"value":0,"assetId":0},{"tileType":1,"value":0,"assetId":0},{"tileType":1,"value":0,"assetId":0},{"tileType":1,"value":0,"assetId":0},{"tileType":1,"value":0,"assetId":0},{"tileType":1,"value":0,"assetId":0}],[{"tileType":1,"value":0,"assetId":0},{"tileType":1,"value":0,"assetId":0},{"tileType":1,"value":0,"assetId":0},{"tileType":1,"value":0,"assetId":0},{"tileType":1,"value":0,"assetId":0},{"tileType":1,"value":0,"assetId":0},{"tileType":1,"value":0,"assetId":0},{"tileType":1,"value":0,"assetId":0},{"tileType":0,"value":0,"assetId":285},{"tileType":1,"value":0,"assetId":0},{"tileType":1,"value":0,"assetId":0},{"tileType":1,"value":0,"assetId":0},{"tileType":1,"value":0,"assetId":0},{"tileType":1,"value":0,"assetId":0},{"tileType":1,"value":0,"assetId":0},{"tileType":1,"value":0,"assetId":0}],[{"tileType":1,"value":0,"assetId":0},{"tileType":1,"value":0,"assetId":0},{"tileType":1,"value":0,"assetId":0},{"tileType":1,"value":0,"assetId":0},{"tileType":1,"value":0,"assetId":0},{"tileType":1,"value":0,"assetId":0},{"tileType":1,"value":0,"assetId":0},{"tileType":1,"value":0,"assetId":0},{"tileType":0,"value":0,"assetId":285},{"tileType":1,"value":0,"assetId":0},{"tileType":1,"value":0,"assetId":0},{"tileType":1,"value":0,"assetId":0},{"tileType":1,"value":0,"assetId":0},{"tileType":1,"value":0,"assetId":0},{"tileType":1,"value":0,"assetId":0},{"tileType":1,"value":0,"assetId":0}]],
```

Comment:
just sanity-checking -- is this level change intentional? it looks like `serialized_maze` changed and `parent_level_id` was removed